docs/user_guides/projects/git/configure_git_provider.md

```diff
@@ -18,6 +18,8 @@ Documentation on how to generate a token for the supported Git hosting services
 - [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
 - [BitBucket](https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html)
 
+The token permission should grant access to public and private repositories including read and write access to code and commit statuses.
```

Comment:
This is a prerequisite for the guide. So we should create a new section such as http://127.0.0.1:8000/user_guides/projects/git/clone_repo/#prerequisites and put this text under. The prerequisites are:

* Create an access token for your git provider.
* The token's permissions should grant access to public and private repositories, including read and write access to code and commit statuses.